config/ksql-server.properties

```diff
@@ -20,7 +20,8 @@
 # The default is any IPv4 interface on the machine.
 # NOTE: If set to wildcard or loopback set 'advertised.listener' to enable pull queries across machines
 listeners=http://0.0.0.0:8088
-
+ksql.service.id=test
```

Comment:
This does not look like it's meant to be checked in.

Reply:
😞 thanks